gui.js

```diff
@@ -1552,9 +1552,20 @@ IDE_Morph.prototype.fixLayout = function (situation) {
 IDE_Morph.prototype.setProjectName = function (string) {
     this.projectName = string.replace(/['"]/g, ''); // filter quotation marks
     this.hasChangedMedia = true;
+    this.updateDocumentTitleText();
```

Comment:
I think we should have a single function, not just a statement. The reason is that we will likely need to call this function in at least 2 or 3 places, and we might want to change the display. Secondly, for applications like BettleBlocks or others it will be easier for them to override a short, simple function rather that every instance where a project name might be displayed.